float8_experimental/float8_linear.py

```diff
@@ -148,6 +153,15 @@ def __init__(self, *args, **kwargs):
         # will access the scale when it has ensured that it is on GPU.
         self._float8_tensor_ctor = lambda *args, **kwargs: Float8Tensor(*args, **kwargs)
 
+        if config.allocate_float8_weight_cache_buffers:
+            # this is a buffer to get `to(dtype)` for free
+            # TODO(future): hide this from serialization
+            # TODO(future): force this to stay in float8_e4m3fn
+            self.register_buffer(
```

Comment:
https://github.com/pytorch-labs/float8_experimental/blob/main/float8_experimental/float8_linear.py#L175

Could probably use something like this for the second todo

Reply:
yep, we can fix that in a future PR!